.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -15,6 +15,16 @@ Please fill in as much of the template below as you're able. Feel free to delete
 ### Search terms you've used
 <!-- What search terms have you used to check whether this bug was already reported? -->
 
+### Impacted package
+
+What package(s) of the monorepo is impacted by the bug ?
```

Comment:
Further to Vinnie's comment, maybe this wording is better (and removes the need for '...package(s) is(are)...' :) !)
```suggestion
Which packages do you think might be impacted by the bug ?
```